google/cloud/spanner_v1/database.py

```diff
@@ -734,6 +735,17 @@ def batch(self, request_options=None):
         """
         return BatchCheckout(self, request_options)
 
+    def mutation_groups(self):
```

Comment:
Can we pass the `request_options` here similar to existing `batch()` method?

Reply:
As discussed offline, that would only be needed if we called `batch_write()` on `__exit__`. Since we aren't doing it, the current way is fine.